src/jit/gentree.cpp

```diff
@@ -9943,6 +9943,28 @@ void Compiler::gtDispRegVal(GenTree* tree)
         }
     }
 
+#if FEATURE_MULTIREG_RET
+    if (tree->IsCopyOrReload())
+    {
+        for (int i = 0; i < MAX_RET_REG_COUNT - 1; i++)
+        {
+            regNumber reg = (regNumber)tree->AsCopyOrReload()->gtOtherRegs[i];
+            if (reg == REG_NA)
+            {
+                break;
+            }
+            printf(",%s", compRegVarName(reg));
+        }
+    }
+#endif
+
+#if !defined(LEGACY_BACKEND) && defined(_TARGET_ARM_)
+    if (tree->OperGet() == GT_PUTARG_REG)
+    {
+        printf(",%s", compRegVarName((regNumber)tree->AsMultiRegOp()->gtOtherReg));
```

Comment:
When is gtOtherReg valid? It looks like it is only valid with `GT_PUTARG_REG` when `opts.compUseSoftFP` is true. It seems like you could also display it for `GT_MUL_LONG` for arm.

Maybe there needs to be a:
```
bool OperIsMultiReg()
{
  return (gtOper == GT_MUL_LONG) || ((gtOper == GT_PUTARG_REG) && opts.compUseSoftFP);
}
```